tespy/components/components.py

```diff
@@ -737,3 +737,44 @@ def zeta2_func(self):
             v_o = v_mix_ph(o, T0=self.outl[1].T.val_SI)
             return (self.zeta2.val - (i[1] - o[1]) * np.pi ** 2 /
                     (8 * abs(i[0]) * i[0] * (v_i + v_o) / 2))
+
+    def zeta3_func(self):
+        r"""
+        Calculate residual value of :math:`\zeta_3`-function.
+
+        Returns
+        -------
+        val : float
+            Residual value of function.
+
+            .. math::
+
+                val = \begin{cases}
+                p_{in} - p_{out} & |\dot{m}| < \epsilon \\
+                \frac{\zeta_3}{D^4} - \frac{(p_{3,in} - p_{3,out}) \cdot \pi^2}
+                {8 \cdot \dot{m}_{3,in} \cdot |\dot{m}_{3,in}| \cdot
+                \frac{v_{3,in} + v_{3,out}}{2}} &
+                |\dot{m}| > \epsilon
+                \end{cases}
```

Comment:
This function is orc_evaporator specific. It should be either moved to that component or we must come up with a way, that the zeta_func handles all (zeta, zeta1, zeta2 and zeta3) function calls and apply this to the heat exchanger classes, too.

Reply:
Yes, the specific zeta3 function for the orc_evaporator has been moved to the orc_evaporator class.